### PR TITLE
I2625 - Burden estimates. After i2634 is merged please!

### PR DESCRIPTION
**Pull request by weshinsley**
Spec:
https://github.com/vimc/montagu-api/blob/master/docs/spec/BurdenEstimates.md
Schemas:
https://github.com/vimc/montagu-api/blob/master/docs/schemas/BurdenEstimates.schema.json
https://github.com/vimc/montagu-api/blob/master/docs/schemas/CreateBurdenEstimateSet.schema.json
https://github.com/vimc/montagu-api/blob/master/docs/schemas/BurdenEstimate.csvschema.json
https://github.com/vimc/montagu-api/blob/master/docs/schemas/StochasticBurdenEstimate.csvschema.json

(I'm not implementing the Token-based upload method - not optimal for the montagu-r use case.)
Issue:
https://vimc.myjetbrains.com/youtrack/issue/VIMC-2625

### CODE REVIEW

**Comment by weshinsley** [comment]
The build on travis is failing because of the images in the vignettes...
* creating vignettes ... ERROR
File figure/montagu_user_guide-unnamed-chunk-19-1.png not found in resource path

Running make vignettes locally, the above file does get created in the vignettes/figure folder...

File: R/demographics.R [hunk 1]
```diff
@@ -71,7 +71,7 @@ montagu_demographics_download <- function(touchstone_id, source_code,
 montagu_demographic_data <- function(type_code, touchstone_id,
                                      gender_code = NULL, wide = FALSE,
                                      source_code = NULL, location = NULL) {
-  location <- montagu_location(location)
+
```
Comment (richfitz) [inline R/demographics.R]: this probably should not have been deleted

File: man/Burden-Estimate-Sets.Rd [hunk 2]
```diff
@@ -0,0 +1,17 @@
+% Generated by roxygen2: do not edit by hand
```
Comment (richfitz) [inline man/Burden-Estimate-Sets.Rd]: this looks suspicious - roxygen error?

File: vignettes/montagu_user_guide.Rmd [hunk 3]
```diff
@@ -440,7 +440,7 @@ the full coverage data for all countries. And for some diseases, the historic ef
 in a country that is not required for estimates as present, may have relevance in their model to
 countries that are required.
 
-For those cases, where all countries are required, the `all_countries` optional argument can be used:
+For those cases, where all countries are required, the following optional argument can be used:
```
Comment (richfitz) [inline vignettes/montagu_user_guide.Rmd]: I feel like "following" is less clear here

File: vignettes/montagu_user_guide.Rmd [hunk 4]
```diff
@@ -494,11 +675,11 @@ params
 
 ```
 ##   run_id param_1 param_2
```
Comment (richfitz) [inline vignettes/montagu_user_guide.Rmd]: are these generated randomly? if so `set.seed` will help cut down on noise

File: man/Burden-Estimate-Sets.Rd (hunk 2 above)
Comment (weshinsley) [reply man/Burden-Estimate-Sets.Rd]: Error in the opening comments of burden.R caused this...

File: vignettes/montagu_user_guide.Rmd (hunk 3 above)
Comment (weshinsley) [reply vignettes/montagu_user_guide.Rmd]: Agreed - reverted/improved.

File: vignettes/montagu_user_guide.Rmd (hunk 4 above)
Comment (weshinsley) [reply vignettes/montagu_user_guide.Rmd]: I do `params <- data.frame(run_id = 1:5, param_1 = sample(5), param_2 = sample(5))` 
at the moment...

Comment (weshinsley) [reply vignettes/montagu_user_guide.Rmd]: Have prefixed that block with
```{r include = FALSE}
set.seed(12345)
```

**Pull request merged**